### PR TITLE
Fix Stored XSS vulnerability on Contact Merge labels

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
# Critical Changes

# Changes
We fixed a potential vulnerability to stored XSS on two Contact Merge page labels

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
@lparrott @daniel-fuller Stored XSS fix ready for review.

@ErinWiedemer I tagged you since there is a label change, but there is actually no change to wording - only a changed from single to double quote in an href.

**Comment by npsp-reedestockton** [comment]
Tracing [W-8935651](https://gus.lightning.force.com/lightning/r/a07B00000097Tf9IAE/view)

**Review by andrewyu-salesforce — APPROVED** [review]
Nice work on reusing HtmlOutput

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -6886,7 +6886,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Text to be display before the &apos;See Duplicate Contact&apos; button</shortDescription>
-        <value>See all duplicate Contacts found using matching rules. &lt;a href=&apos;https://powerofus.force.com/IP_Contacts&apos; target=&apos;_blank&apos;&gt;&lt;b&gt; Tell Me More &lt;/b&gt;&lt;/a&gt;</value>
+        <value>See all duplicate Contacts found using matching rules. &lt;a href=&quot;https://powerofus.force.com/IP_Contacts&quot; target=&apos;_blank&apos;&gt;&lt;b&gt; Tell Me More &lt;/b&gt;&lt;/a&gt;</value>
```
Comment (daniel-fuller) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:6889]: @npsp-reedestockton Did this need to be changed? Saw the encoding changed from single to double quote but it's not changed for the `target` attribute

Comment (npsp-reedestockton) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:6889]: > @npsp-reedestockton Did this need to be changed? Saw the encoding changed from single to double quote but it's not changed for the `target` attribute

@daniel-fuller It works fine as is.  The only reason for the encoding change was to pass it through the allow list filter in UTIL_HtmlOutput_CTRL.  The target attribute isn't considered in the allow list filter so it didn't need to change.  If it seems to weird to have the mix, we could change it to be consistent, but it isn't necessary.

Comment (lparrott) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:6889]: Any reason for this change? It works fine, I'm just curious.

**Review by lparrott — APPROVED** [review]

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (lparrott) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:6889]: I had the same question, seems to work fine so no reason to change it.

Comment (npsp-reedestockton) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml]: @daniel-fuller Consistency is a good thing.  I made the change you suggested.

**Review by daniel-fuller — APPROVED** [review]

**Review by ErinWiedemer — COMMENTED** [review]
LGTM!

**Pull request merged**